routes/mandatees-decisions.ts

```diff
@@ -3,15 +3,29 @@ import Router from 'express-promise-router';
 import { Request, Response } from 'express';
 
 import { Changeset } from '../types';
-import { handleDeltaChangeset } from '../controllers/mandatees-decisions';
+import { handleTriplesForMandatarisSubject } from '../controllers/mandatees-decisions';
 import { ProcessingQueue } from '../services/processing-queue';
+import {
+  getSubjectsOfType,
+  TERM_MANDATARIS_TYPE,
+} from '../data-access/mandatees-decisions';
 
 const mandateesDecisionsRouter = Router();
-const todo = new ProcessingQueue();
+export const mandatarisQueue = new ProcessingQueue();
 
 mandateesDecisionsRouter.post('/', async (req: Request, res: Response) => {
-  const changeSets: Changeset[] = req.body;
-  todo.addToQueue(async () => await handleDeltaChangeset(changeSets));
+  const changesets: Changeset[] = req.body;
+  const insertTriples = changesets
+    .map((changeset: Changeset) => changeset.inserts)
+    .flat();
+  const mandatarisSubjects = await getSubjectsOfType(
+    TERM_MANDATARIS_TYPE,
+    insertTriples,
+  );
+
+  mandatarisQueue.setMethodToExecute(handleTriplesForMandatarisSubject);
+  mandatarisQueue.addToQueue(mandatarisSubjects);
+  mandatarisQueue.moveManualQueueToQueue();
```

Comment:
I'm a bit confused about the manual queue. Am I right to assume you use the manual queue, so you only allow the queue to process these mandataris subjects if new triples come in?